mdtraj/tests/test_load.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# MDTraj: A Python Library for Loading, Saving, and Manipulating
+#         Molecular Dynamics Trajectories.
+# Copyright 2012-2013 Stanford University and the Authors
+#
+# Authors: Robert McGibbon
```

Comment:
Feel free to put your name here and update the copyright date and/or remove this whole header.